src/commands/publish.ts

```diff
@@ -235,4 +235,26 @@ const rollback = async (): Promise<void> => {
     }
 };
 
-export { publish, update, release, rollback, newMajorVersion };
+const rollbackDetails = (): {
+    current: number;
+    target: number;
+    name: string;
+} => {
+    validateFilesExistOrExit();
+    validateBlockExistOrExit();
+
+    const { activeVersion, displayName } = readBlockSettingsFile(
+        BLOCK_SETTINGS_FILE
+    );
+
+    const version = activeVersion || 1;
+    const target = version - 1;
```

Comment:
This is not correct. Rollback do not roll from major to major. It rollsback from minor to minor, until a major is reached. 

1
1.1
1.2
1.3  <- current

Rollback -> Current = 1.2 -> Rollback -> Current = 1.1 -> Rollback -> Current = 1 -> Rollback -> Error. You reached the major release.

Reply:
That makes sense. When I read the block settings file, if the block was on a minor version, would it look like this?
activeVersion: 1.2

Reply:
No, it will always be 1. We don't assign numbers to minor releases. We have many minor, with one of them set as active.